CRM/Contribute/Import/Parser/Contribution.php

```diff
@@ -593,8 +593,9 @@ public function formatDateFields(&$params) {
    * CRM_Contribute_Import_Parser_ContributionTest.
    *
    * @param array $params
+   * @param array $formatted
    */
-  public function formatInput(&$params) {
+  public function formatInput(&$params, &$formatted) {
```

Comment:
any reason why we need this to be passed by reference @eileenmcnaughton @sluc23

Reply:
I think it was a quick way to address the regression but we can fix it better - looking

Reply:
ok i see needs to be as the formatCustomDate one does it too so we need to tranverse all the way back up